### DISCUSSION TITLE
Extended digital resolution

### DISCUSSION THREAD

**Discussion by JensHoRi** [jomjol/AI-on-the-edge-device#988] [Ideas]
Actual there is an extended resolution for analog configurable.

As the new version has sich great reading of inter-digits a configurable extended resolution digital would be great.

not needed for water meters, but great for electricity:)

**Comment by jomjol** [comment]
This parameter is also working for the digits (if you are using cnns with inter-digits reading.

![grafik](https://user-images.githubusercontent.com/30766535/188934699-d4ae590d-c4aa-4ad0-b920-38456863e14a.png)

**Comment by JensHoRi** [comment]
Crazy, first try failed, now it worked. Thanks
